heartbeat/tests/system/test_telemetry.py

```diff
@@ -47,7 +48,7 @@ def test_telemetry(self):
                 }
             })
 
-            tcp_hosts = ["localhost:8185", "localhost:12345"]
+            tcp_hosts = ["localhost:123", "localhost:456"]
```

Comment:
Just curious, can these ports affect the result of this test? If not, why changing them?

Reply:
I just wanted them to look more `fake` since they actually can't affect the results.